Card Component/card.css

```diff
@@ -0,0 +1,146 @@
+*{
+    padding: 0px;
+    margin: 0px;
+    font-size: 62.5%;
+    box-sizing: border-box;
+}
+
+body{
+  
+}
+
+.main{
+    display: flex;
+    flex-direction: column;
+    gap : 2rem;
+    justify-content: center;
+    align-items: center;
+    background-color: #686672;
```

Comment:
You can create variables for the color.

Reply:
Hi Mahesh,
Added card with dismiss and card with badge, thanks for pointing out, I missed those earlier.
Also added color variables and padding on card content.

![image](https://user-images.githubusercontent.com/81914134/152628748-3aaa135c-b44e-4fdb-914d-1ee7c762c187.png)
![image](https://user-images.githubusercontent.com/81914134/152628754-f2587019-d02a-4109-8ada-ec2ec58f0ca3.png)